### PR TITLE
Update docker.md

### PR DESCRIPTION
**Pull request by andreaturli**

### CODE REVIEW

File: guides/docker.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ This involves [dockerizing an SSH daemon service](https://docs.docker.com/exampl
 // get a context with docker that offers the portable ComputeService api
 ComputeServiceContext context = ContextBuilder.newBuilder("docker")
                       .credentials(email, password)
+                      .endpoint("http://your-docker-engine-host:2375")
```
Comment (andreaturli) [inline guides/docker.md]: yes as they are officially listed [here](http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=docker)

**Comment by jclouds-commentator** [comment]
Go to http://1fe31db7584901aebe8a-dae5426beae6a9fc68a207722950803e.r58.cf5.rackcdn.com/ to review your changes.

**Comment by andreaturli** [comment]
@demobox I've addressed your questions. Are we good to merge that now?

**Comment by demobox** [comment]
> Are we good to merge that now?

Yes, fine by me. Thanks for the ping!

**Pull request merged**